src/index.ts

```diff
@@ -9,17 +8,12 @@ import { DynamoService, IDynamoService } from './IDynamoService';
 export { IDynamoOperation, IDynamoService, DynamoService, DynamoFactory, DynamoServices };
 
 export interface UseDynamoConfiguration {
-  core?: DynamoDBClientConfig;
-  documentTranslation?: TranslateConfig;
+  core?: DynamoDB.Types.ClientConfiguration;
 }
 
 export const useDynamo: (options: UseDynamoConfiguration) => IServiceModule =
```

Comment:
this could turn into

`useDynamo: (options: DynamoDB.Types.ClientConfiguration)...`